packages/@react-aria/button/docs/useButton.mdx

```diff
@@ -0,0 +1,116 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/button';
+import {HeaderInfo, FunctionAPI} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {useButton} from '@react-aria/button';
+import {useRef} from 'react';
+```
+
+# useButton
+
+<p>{docs.exports.useButton.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useButton']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-practices/#button'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useButton} links={docs.links} />
```

Comment:
Well, because there are more than just props being sent to the function. It's not a component. You can still click on the type in the function API to expand it.